Client/game_sa/CStreamingSA.cpp

```diff
@@ -431,6 +431,8 @@ void CStreamingSA::RemoveArchive(unsigned char ucArchiveID)
 
 bool CStreamingSA::SetStreamingBufferSize(uint32 numBlocks)
 {
+    numBlocks &= ~1; // Make it be even [Otherwise it can't be split in half properly]
+    
```

Comment:
Does it make less blocks?

Reply:
Yes. if `numBlocks` is 41 it allocates only 40 (42 is expected)

Reply:
```suggestion
    numBlocks += numBlocks % 2; // Make sure number is even by "rounding" it upwards. [Otherwise it can't be split in half properly]
```